### ISSUE TITLE
npm-groovy-lint indenting closing brace incorrectly 

### ISSUE THREAD

**Issue by wheelerlaw** [nvuillam/npm-groovy-lint#153]
So lets say I have this function:

```groovy
def testFunction() {
    testClosure(["first item", "second item",
                 "newline item"]) {
        println("hello world")
    }
}
```

When I run `npm-groovy-lint`, it moves the closing `}` for `testClosure` into a very weird and wrong spot:

```groovy
def testFunction() {
    testClosure(["first item", "second item",
                 "newline item"]) {
        println("hello world")
                 }
}
```

Ideally it would either do 1) do nothing, 2) collapse the closure parameters onto one line (if there is space): 

```groovy
def testFunction() {
    testClosure(["first item", "second item", "newline item"]) {
        println("hello world")
    }
}
```

Or 3) bring the parameters onto a newline and close the parameters with the closing square bracket and opening brace on a new line:

```groovy
def testFunction() {
    testClosure([
        "first item",
        "second item",
        "newline item"
    ]) {
        println("hello world")
    }
}
```

In anycase, messing with the closing brace is definitely wrong.

**Comment by nvuillam** [comment]
npm-groovy-lint depends on CodeNarc for most of its corrections... and CodeNarc V2 Indentation rule sometime provides different "good" position for brackets... and as far as I know there are no way to guess the appropriate one :/

Let's hope that in a next version it will be solved, so npm-groovy-lint can benefit from it :)

**Comment by nvuillam** [comment]
@wheelerlaw cf https://github.com/CodeNarc/CodeNarc/issues/553

**Comment by wheelerlaw** [comment]
I assume that's also probably the root casue of #152 ?

**Comment by nvuillam** [comment]
Yes it is :(
